themes/bootstrap3/js/common.js

```diff
@@ -549,6 +549,131 @@ function bulkFormHandler(event, data) {
+          gapFiller.classList.remove("hidden");
+          let parentStyle = window.getComputedStyle(stickyContainer.parentNode, null);
+          let parentBoundingClientRect = stickyContainer.parentNode.getBoundingClientRect();
+          stickyContainer.style.top = (currentOffset - 1) + "px";
+          stickyContainer.style.marginLeft = parentBoundingClientRect.left + "px";
+          stickyContainer.style.marginRight = (window.screen.width - parentBoundingClientRect.right) + "px";
+          stickyContainer.style.borderLeft = parentStyle.borderLeft;
+          stickyContainer.style.borderRight = parentStyle.borderRight;
+          stickyContainer.style.paddingLeft = parentStyle.paddingLeft;
+          stickyContainer.style.paddingRight = parentStyle.paddingRight;
+          stickyContainer.style.width = parentBoundingClientRect.width + "px";
+          stickyContainer.style.zIndex = 10 + count - num;
+          gapFiller.style.width = stickyElementStyle.width;
+          gapFiller.style.marginLeft = stickyElementStyle.marginLeft;
+          gapFiller.style.borderLeft = stickyElementStyle.borderLeft;
+          gapFiller.style.paddingLeft = stickyElementStyle.paddingLeft;
+          currentOffset -= 1;
+        } else {
+          stickyContainer.classList.remove("sticky");
+          placeholder.classList.add("hidden");
+          gapFiller.classList.add("hidden");
+          stickyContainer.style.top = "";
+          stickyContainer.style.marginLeft = "0";
+          stickyContainer.style.marginRight = "0";
+          stickyContainer.style.borderLeft = "0";
+          stickyContainer.style.borderRight = "0";
+          stickyContainer.style.paddingLeft = "0";
+          stickyContainer.style.paddingRight = "0";
+          stickyContainer.style.width = "";
+          stickyContainer.style.zIndex = "";
+        }
+        currentOffset += stickyContainer.offsetHeight;
+        num += 1;
+      });
+  }
+
+  stickyElements.forEach(
+    (stickyElement) => {
+      let placeholder = document.createElement('div');
+      placeholder.classList.add('sticky-placeholder', 'hidden');
+      stickyElement.parentNode.insertBefore(placeholder, stickyElement);
+
+      let container = document.createElement('div');
+      container.classList.add('sticky-container');
+      stickyElement.parentNode.insertBefore(container, stickyElement);
+
+      stickyElement.previousSibling.insertAdjacentElement('beforeEnd', stickyElement);
+
+      let gapFiller = document.createElement('div');
+      gapFiller.classList.add('sticky-gap-filler', 'hidden');
+      stickyElement.parentNode.insertBefore(gapFiller, stickyElement);
+
+      setPlaceholderStyle(stickyElement);
+      stickyElement.parentNode.style.backgroundColor = getInheritedBackgroundColor(stickyElement.parentNode);
+      stickyElement.previousSibling.style.backgroundColor = getInheritedBackgroundColor(stickyElement);
+    }
+  );
+  handleStickyElements();
+
+  window.addEventListener("resize", () => {
```

Comment:
Should we skip setting up these event listeners when no sticky elements are found, to avoid doing unnecessary work?

Reply:
Returning early now if no elements are found.